larq_compute_engine/tests/qemu_test.bzl

```diff
@@ -35,10 +35,11 @@ def lce_qemu_test_suite(
         # Finally create a new sh_test target
         native.sh_test(
             name = sh_name,
-            size = "medium",
+            size = "large",
```

Comment:
I've made this "large" because otherwise the 'big' kernel tests which don't run on CI timeout.